### PR TITLE
samples: alexa_gadgets: Remove alexa gadgets sample

### PR DESCRIPTION
**Pull request by nordic-auko**
Remove alexa gadgets sample application and associated files,
due to amazon having "paused support for third-party device makers
working with Alexa Gadgets" (see Alexa Gadgets Toolkit website
for full statement and future updates).

Signed-off-by: Audun Korneliussen <audun.korneliussen@nordicsemi.no>

### CODE REVIEW

**Comment by nordic-auko** [comment]
I edited the v1.4.0 release notes to avoid a doc ref because of a reference to documentation that has now been removed. Not sure if this is the best way to solve this issue

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -147,6 +147,8 @@ Bluetooth samples
 
 * Added :ref:`peripheral_ams_client` sample.
 
+* Removed alexa_gadgets Bluetooth sample due to Amazon pausing support for the Gadgets ecosystem.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:150]:
```suggestion
* Removed Peripheral Alexa Gadgets Bluetooth sample due to Amazon pausing support for the Gadgets ecosystem.
```
Please add also the removal of the Alexa Gadgets service library.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (nordic-auko) [reply doc/nrf/releases/release-notes-changelog.rst]: Done. Reference to the library in the old release notes was removed already, as this caused a doc build failure.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -147,6 +147,8 @@ Bluetooth samples
 
 * Added :ref:`peripheral_ams_client` sample.
 
+* Removed Peripheral Alexa Gadgets Bluetooth sample and Alexa Gadgets Service library due to Amazon pausing support for the Gadgets ecosystem.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:150]: I believe the entry about the removal of the library needs to go under Libraries -> Bluetooth libraries and services, not combined to this entry.

**Review by peknis — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

**Pull request merged**